x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/runtime_mappings/types.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  RUNTIME_FIELD_TYPES,
+  RuntimeField,
+  RuntimeMappings,
+  RuntimeType,
+} from '../../../../../../../common/types/fields';
+
+export const isPopulatedObject = <T = Record<string, unknown>>(arg: unknown): arg is T => {
+  return typeof arg === 'object' && arg !== null && Object.keys(arg).length > 0;
+};
+
+export function isRuntimeField(arg: unknown): arg is RuntimeField {
```

Comment:
I thine we can remove these and use exports from `ml/common/util/runtime_field_utils.ts`